src/PaymentStates.php

```diff
@@ -25,10 +25,17 @@
 
 final class PaymentStates
 {
+    /**
+     * Order states
```

Comment:
I think order states and payment states are not the same and this may be confusing for other frameworks or custom implementations